Update scripts.md per style guide (#519)
bigcommerce/dev-docs: docs/api-docs/storefront/scripts-overview.md

<div class="otp" id="no-index">
	<h3> On This Page </h3>
	<ul>

>     	<li><a href="#scripts_script-manager-partner-guidelines">Script manager API partner guidelines</a></li>

Reviewer: ```suggestion
    	<li><a href="#scripts_script-manager-partner-guidelines">Script Manager API partner guidelines</a></li>

```

        <li><a href="#script_prerequisites">Prerequisites</a></li>
        <li><a href="#scripts_upgrades-installation">Upgrades and installation</a></li>
        <li><a href="#scripts_fixing-missing-scripts">Fixing missing scripts</a></li>
        <li><a href="#scripts_scripts-notes">Notes</a></li>
        <li><a href="#scripts_scripts-visibility">Visibility locations </a></li>
	</ul>
</div>

> The BigCommerce Scripts API gives developers the ability to inject scripts into a store's template files programmatically. This ability means that app and integrations can insert scripts into a user’s storefront without requiring the user to paste a snippet of code into their control panel manually. There are many use cases for this powerful API, for example:

Reviewer: ```suggestion
The BigCommerce Scripts API gives developers the ability to inject scripts into a store's template files programmatically. This ability means that apps and integrations can insert scripts into a user’s storefront without requiring the user to paste a snippet of code into their control panel manually. You can insert many types of scripts using this API, including the following:

```

* inserting analytics scripts
* inserting single-click app scripts
* inserting live chat and support plugins
* inserting theme extensions or connector apps

> 

Reviewer: Remove the leading "inserting" in the list items above to avoid redundancy. 
Bullet points should also be followed by capital letters

> ## Script manager API partner guidelines

Reviewer: ```suggestion
## Script Manager API partner guidelines

```

<div class="otp" id="no-index">
	<h3> On This Page </h3>
	<ul>
    	<li><a href="#scripts_script-manager-partner-guidelines">Script Manager API partner guidelines</a></li>

        <li><a href="#script_prerequisites">Prerequisites</a></li>
        <li><a href="#scripts_upgrades-installation">Upgrades and installation</a></li>
        <li><a href="#scripts_fixing-missing-scripts">Fixing missing scripts</a></li>
        <li><a href="#scripts_scripts-notes">Notes</a></li>
        <li><a href="#scripts_scripts-visibility">Visibility locations </a></li>
	</ul>
</div>

The BigCommerce Scripts API gives developers the ability to inject scripts into a store's template files programmatically. This ability means that apps and integrations can insert scripts into a user’s storefront without requiring the user to paste a snippet of code into their control panel manually. You can insert many types of scripts using this API, including the following:

> * inserting analytics scripts

Reviewer: ```suggestion
* Analytics scripts

```

> * inserting single-click app scripts

Reviewer: ```suggestion
* Single-click app scripts

```

> * inserting live chat and support plugins

Reviewer: ```suggestion
* Live chat and support plugins

```

> * inserting theme extensions or connector apps

Reviewer: ```suggestion
* Theme extensions or connector apps

```

Because of this, you’ll need to check whether a user is running Stencil to determine if their store supports the Script Manager API. To do this, use the [Get Store Information API](https://developer.bigcommerce.com/api/v2/#store-information-reference) endpoint and check the `stencil_enabled` flag. (requires [store_v2_information_read_only scope](https://developer.bigcommerce.com/api/#oauth-scopes)).

## Prerequisites
This article contains detailed information about the Scripts API and assumes you're familiar with BigCommerce App Installation process. For more information on the app installation process, see [App Installation](/api-docs/getting-started/building-apps-bigcommerce/building-apps#building-apps_installation-update-sequence)

## Upgrades and installation

### Installing an app on Stencil

We recommend inserting your scripts immediately after receiving the POST response during the Auth Callback flow for apps installed on Stencil stores. Add your scripts using the Create Script endpoint of the Script Manager API.

> We highly recommend leaving the auto_uninstall flag set to true, so that your app will properly clean itself up when uninstalled. Find more information about this flag <a href="#script_object-properties">here</a>.

Reviewer: ```suggestion
We highly recommend leaving the `auto_uninstall` flag set to true, so that your app will properly clean itself up when uninstalled. Find more information about this flag <a href="#script_object-properties">here</a>.

```
The link here doesn't work

#### Upgrading Existing Apps on Stencil

If you already released your app, you may have existing users with pasted-in code somewhere in their control panel. There are a few possible ways to deal with this situation.

> 1.) __Do nothing__

Reviewer: ```suggestion
__Do nothing__

```

If you don’t need to upgrade existing merchants to the Script Manager API, it’s perfectly fine to leave them. You’ll need to maintain documentation for the manual uninstallation process so that merchants with older installations know what to clean up. You won’t be able to update or add scripts for these users automatically. 

> 2.) __Prompt merchants to upgrade__

Reviewer: ```suggestion
 __Prompt merchants to upgrade__

```

If you want your merchants to update to the new Script Manager over time, you can prompt them to do so on your app’s landing page. You’ll need to keep track of which of your merchants are not using the Script Manager to know whether you need to display this prompt. Make sure to check that they are running Stencil (via the Get Store Information endpoint) before prompting them to upgrade.

Once a user decides to upgrade, you can walk them through removing the old pasted-in code in their control panel, and then, once the merchant clicks a button, install your scripts through the Script Manager API.

> 3.) __Silently upgrade to the Script Manager API__

Reviewer: ```suggestion
__Silently upgrade to the Script Manager API__

```
